File: cigarParty/CigarParty.java
package cigarParty;

public class CigarParty {
    //Retorne true se o número de "cigars" estiver entre 40 e 60, inclusive, a menos que o atributo "isWeekend" seja true,
    //nesse caso não há limite superior.

    public boolean cigarParty(int cigars, boolean isWeekend) {
        if (isWeekend && cigars >= 40) return true;
        else return (cigars >= 40 && cigars <= 60);
    }
}

File: Python/Logic-1/cigar_party.py
"""
Retorne true se "cigars" estiver entre 40 e 60, inclusive, a menos que "is_weekend"
seja true, nesse caso não há limite superior.
"""


def cigar_party(cigars, is_weekend):
    if cigars >= 40:
        return cigars <= 60 or is_weekend
    return False


print(cigar_party(70, True))
